tools/changelog.main.kts

```diff
@@ -0,0 +1,298 @@
+/**
+ * Script for creating a changelog. Call
+ * ```
+ * kotlin changelog.main.kts v1.6.0-rc02 release/1.6.0
+ * ```
+ * where v1.6.0-rc02 - the first commit
+ * where release/1.6.0 - the last commit
+ */
+
+/**
+ * Run from command line:
+ * 1. Download https://github.com/JetBrains/kotlin/releases/tag/v1.9.22 and add `bin` to PATH
+ * 2. Call `kotlin <fileName>`
+ *
+ * Run from IntelliJ:
+ * 1. Right click on the script
+ * 2. More Run/Debug
+ * 3. Modify Run Configuration...
+ * 4. Clear all "Before launch" tasks (you can edit the system-wide template as well)
+ * 5. OK
+ */
+
+@file:Repository("https://repo1.maven.org/maven2/")
+@file:DependsOn("com.google.code.gson:gson:2.10.1")
+
+import com.google.gson.Gson
+import java.io.File
+import java.io.IOException
+import java.net.URL
+import java.util.concurrent.TimeUnit
+
+val firstCommit = args.getOrNull(0) ?: error("Please call this way: kotlin changelog.main.kts <firstCommit> <lastCommit>")
+val lastCommit = args.getOrNull(1) ?: error("Please call this way: kotlin changelog.main.kts <firstCommit> <lastCommit>")
+val token = args.getOrNull(2)
+
+val sectionOrder = listOf(
+    "Highlights",
+    "Known issues",
+    "Breaking changes",
+    "Features",
+    "Fixes",
+    null
+)
+
+val subsectionOrder = listOf(
+    "Multiple Platforms",
+    "iOS",
+    "Desktop",
+    "Web",
+    "Resources",
+    "Gradle Plugin",
+    "Lifecycle",
+    "Navigation",
+    null
+)
+
+if (token == null) {
+    println("To increase the rate limit, specify token (https://github.com/settings/tokens): kotlin changelog.main.kts <firstCommit> <lastCommit> TOKEN")
+}
+
+val entries = entriesForRepo("JetBrains/compose-multiplatform-core") +
+        entriesForRepo("JetBrains/compose-multiplatform")
+
+println("\n# CHANGELOG")
+
+println(
+    buildString {
+        append("_Changes since ${commitToVersion(firstCommit)}_\n")
+        append("\n")
+
+        entries
+            .sortedBy { it.sectionOrder() }
+            .groupBy { it.sectionName() }
+            .forEach { (section, sectionEntries) ->
+                appendLine("## $section")
+
+                sectionEntries
+                    .sortedBy { it.subsectionOrder() }
+                    .groupBy { it.subsectionName() }
+                    .forEach { (subsection, subsectionEntries) ->
+                        appendLine("### $subsection")
+                        subsectionEntries.forEach {
+                            appendLine(it.format())
+                        }
+                        appendLine()
+                    }
+            }
+    }
```

Comment:
I guess we can parse configs and automatically add

```suggestion
        appendLine("## Dependencies\n")
        appendLine("- Gradle Plugin `org.jetbrains.compose`, version `${version}`. Based on Jetpack Compose libraries:")
        // TBD
        appendLine("___")
    }
```

Reply:
Fixed